src/local/application-dataflow-keycloak.yaml

```diff
@@ -0,0 +1,50 @@
+spring:
+  cloud:
+    dataflow:
+      security:
+        authorization:
+          provider-role-mappings:
+            keycloak:
+              map-group-claims: false
+              map-oauth-scopes: false
+              role-mappings:
+                ROLE_VIEW: dataflow_view
+                ROLE_CREATE: dataflow_create
+                ROLE_MANAGE: dataflow_manage
+                ROLE_DEPLOY: dataflow_deploy
+                ROLE_DESTROY: dataflow_destroy
+                ROLE_MODIFY: dataflow_modify
+                ROLE_SCHEDULE: dataflow_schedule
+  security:
+    oauth2:
+      client:
+        provider:
+          keycloak:
+            issuer-uri: 'http://localhost:8080/realms/dataflow'
+            jwk-set-uri: 'http://localhost:8080/realms/dataflow/protocol/openid-connect/certs'
+            token-uri: 'http://localhost:8080/realms/dataflow/protocol/openid-connect/token'
+            user-info-uri: 'http://localhost:8080/realms/dataflow/protocol/openid-connect/userinfo'
+            user-name-attribute: 'user_name'
+            authorization-uri: 'http://localhost:8080/realms/dataflow/protocol/openid-connect/auth'
+        registration:
+          keycloak:
+            redirect-uri: '{baseUrl}/login/oauth2/code/{registrationId}'
+            client-id: 'dataflow'
+            client-name: 'dataflow'
+            client-secret: '090RucamvekrMLyGHMr4lkHX9xhAlsqK'
+            provider: 'keycloak'
+            authorization-grant-type: 'authorization_code'
+#            client-authentication-method: # unsure of value
+            scope:
+              - openid
+              - roles
+      resourceserver:
+        opaquetoken:
+          introspection-uri: http://localhost:8080/realms/dataflow/protocol/openid-connect/token/introspect
+          client-id: 'dataflow'
+          client-secret: '090RucamvekrMLyGHMr4lkHX9xhAlsqK'
```

Comment:
We need a note here to tell folks to use their own secret here.

Reply:
That secret is specific to the realm we import when using the container.